contrib/docker/superset_config.py

```diff
@@ -45,12 +45,18 @@ def get_env_variable(var_name, default=None):
 
 REDIS_HOST = get_env_variable('REDIS_HOST')
 REDIS_PORT = get_env_variable('REDIS_PORT')
-
+REDIS_PASSWORD = get_env_variable('REDIS_PASSWORD')
+REDIS_DBNUMBER = get_env_variable('REDIS_DBNUMBER')
 
 class CeleryConfig(object):
     BROKER_URL = 'redis://%s:%s/0' % (REDIS_HOST, REDIS_PORT)
     CELERY_IMPORTS = ('superset.sql_lab', )
-    CELERY_RESULT_BACKEND = 'redis://%s:%s/1' % (REDIS_HOST, REDIS_PORT)
+    if REDIS_DBNUMBER == '' or REDIS_DBNUMBER is None:
```

Comment:
Logic (`if`) in class definition seems like a bad idea. How about setting `CELERY_RESULT_BACKEND` above in module scope?